src/coreclr/tools/aot/Directory.Build.targets

```diff
@@ -1,6 +1,6 @@
 <Project>
-  <Import Project="../Directory.Build.props" />
   <PropertyGroup>
     <IsTrimmable Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">true</IsTrimmable>
   </PropertyGroup>
+  <Import Project="../Directory.Build.targets" />
```

Comment:
Please move the import above the custom property as we do in other Directory.Build.targets files: https://github.com/dotnet/runtime/blob/64243bbf5e9ee53c0c4c5678f2cd8c7f1c9b4f6f/src/libraries/System.Runtime.Caching/Directory.Build.targets#L2